legacy_publisher/dita_publisher.py

```diff
@@ -124,6 +124,11 @@ def publish_country_regions(regions=None, stcountries=None, nstcountries=None):
         print("export_dita :", export_dita)
         write_dita_doc(root, export_dita)
 
+        if(dita_ot != None):
+            xml_file = abspath(export_dita)
+            dtd_file = dita_ot+'/plugins/org.oasis-open.dita.v1_2/dtd/technicalContent/dtd/topic.dtd'
```

Comment:
we should make the path to the topic.dtd a constant. We're using it in two places so far, we're bound to use it in more places too.